assets/js/editor-components/product-attribute-term-control/index.tsx

```diff
@@ -65,7 +65,7 @@ const ProductAttributeTermControl = ( {
 			},
 		];
 
-		if ( ! item.breadcrumbs.length ) {
+		if ( ! item?.breadcrumbs?.length ) {
```

Comment:
If `breadcrumbs` always exists, then we need to update this and other optional chaining calls.